ipn/localapi/localapi.go

```diff
@@ -586,6 +616,29 @@ func (h *Handler) serveLoginInteractive(w http.ResponseWriter, r *http.Request)
 	return
 }
 
+func (h *Handler) serveStart(w http.ResponseWriter, r *http.Request) {
+	if !h.PermitWrite {
+		http.Error(w, "access denied", http.StatusForbidden)
+		return
+	}
+	if r.Method != "POST" {
+		http.Error(w, "want POST", 400)
```

Comment:
```suggestion
		http.Error(w, "want POST", http.StatusBadRequest)
```

Reply:
there are 25 occurrences of this in that file. Let's keep it for now and decide how to clean them all up in the same way in its own commit. Consistency is better for now.

Reply:
SGTM